src/site/stages/build/process-cms-exports/schemas/input/node-full_width_banner_alert.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    created: { $ref: 'GenericNestedString' },
+    changed: { $ref: 'GenericNestedString' },
+    metatag: { $ref: 'RawMetaTags' },
+    path: { $ref: 'RawPath' },
+    field_administration: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_alert_dismissable: { $ref: 'GenericNestedBoolean' },
+    field_alert_email_updates_button: { $ref: 'GenericNestedBoolean' },
+    field_alert_find_facilities_cta: { $ref: 'GenericNestedBoolean' },
+    field_alert_inheritance_subpages: { $ref: 'GenericNestedBoolean' },
+    field_alert_operating_status_cta: { $ref: 'GenericNestedBoolean' },
+    field_alert_type: { $ref: 'GenericNestedString' },
+    field_banner_alert_computdvalues: { $ref: 'GenericNestedString' },
```

Comment:
This is, in fact, the correct field name, despite the two typos. :facepalm: 
![image](https://user-images.githubusercontent.com/12970166/89062150-cb33c700-d31a-11ea-90a2-97bb486d427f.png)

Reply:
I initially thought It was my typo